numbagg/moving.py

```diff
@@ -6,48 +6,47 @@
 
 @guvectorize(
     [(float64[:], float64, float64[:])],
-    signature='(n),()->(n)',
+    signature="(n),()->(n)",
     nopython=True,
-    target='parallel',
+    target="parallel",
 )
 def ewm_nanmean(a, com, out):
 
     N = len(a)
     if N == 0:
         return
 
-    alpha = 1. / (1. + com)
-    old_wt_factor = 1. - alpha
-    new_wt = 1.
+    alpha = 1.0 / (1.0 + com)
+    old_wt_factor = 1.0 - alpha
+    new_wt = 1.0
 
     weighted_avg = a[0]
-    is_observation = (weighted_avg == weighted_avg)
+    is_observation = weighted_avg == weighted_avg
     nobs = int(is_observation)
     out[0] = weighted_avg
-    old_wt = 1.
+    old_wt = 1.0
 
     for i in range(1, N):
         cur = a[i]
-        is_observation = (cur == cur)
+        is_observation = cur == cur
         nobs += int(is_observation)
         if weighted_avg == weighted_avg:
             if is_observation:
                 old_wt *= old_wt_factor
 
                 # avoid numerical errors on constant series
                 if weighted_avg != cur:
-                    weighted_avg = ((old_wt * weighted_avg) +
-                                    (new_wt * cur)) / (old_wt + new_wt)
+                    weighted_avg = ((old_wt * weighted_avg) + (new_wt * cur)) / (
+                        old_wt + new_wt
+                    )
```

Comment:
this is the only line that looks actively bad to me with this new formatting

Reply:
I agree